index.html

```diff
@@ -0,0 +1,202 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="main.css" />
+    <title>Singolo</title>
+  </head>
+  <body id="home">
+
+    <div class="contain_header">

```

Comment:
1. You don't need to use additional div in this case. <header> is the save tag as div but only with semantic meaning.
It is better to write <header class="contain_header">
2. In class name use nouns(+adjectives), but not verb (contain - is incorrect, container_header - it's better)